client/src/Component/views/Landing.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Banner from '../Common/Banner/index';
```

Comment:
I think you need to create a file called index inside Banner folder. I think you forget to raise it here.

Reply:
This is so strange! I have an index file on my local branch and everything works. On my Github branch there is no index file, but there is one in this pull request just below this file. There does seem to be something weird with the folder capitalization in the one below though, so maybe that is having an effect ...